models/users.js

```diff
@@ -56,6 +58,25 @@ const addOrUpdate = async (userData, userId = null) => {
   }
 };
 
+const addJoinData = async (userData, username) => {
+  try {
+    await joinModel.doc(username).set(userData);
```

Comment:
Hey ritik don't use username for doc id if the user changes his username, things might break